.github/workflows/project.yml

```diff
@@ -0,0 +1,17 @@
+name: Add PR to project
+
+on:
+  pull_request:
+    types:
+      - opened
```

Comment:
Don't we get draft PR with this?

Reply:
yea would be added, then we will label as in-progress